tests/Kernel/ShortTitleTokenReplaceTest.php

```diff
@@ -37,10 +38,11 @@ class ShortTitleTokenReplaceTest extends RdfKernelTestBase {
   /**
    * {@inheritdoc}
    */
-  protected function setUp() {
+  protected function setUp(): void {
```

Comment:
Should we specify return type in overridden method?

Reply:
I think do yes, phpstorm already complains about it otherwise.

Reply:
We can specify the same, we can't change though.